#iZ XXXXX
not yet raised

RCS file: /cvs/util/extensions/source/nsplugin/source/makefile.mk,v
retrieving revision 1.3
diff -u -r1.3 makefile.mk
--- extensions/source/nsplugin/source/makefile.mk	3 Sep 2004 11:41:22 -0000	1.3
+++ extensions/source/nsplugin/source/makefile.mk	4 Oct 2004 15:42:46 -0000
@@ -77,6 +77,9 @@
 .IF "$(GUI)"=="UNX"
 .IF "$(OS)"=="LINUX"
 INC+= -DNP_LINUX
+CFLAGS+=`pkg-config --cflags gtk+-2.0`
+.ELIF "$(OS)"=="FREEBSD"
+CFLAGS+=`pkg-config --cflags gtk+-2.0`
 .ENDIF
 .ENDIF
 .IF "$(GUI)"=="WNT"
@@ -93,8 +96,12 @@
 .IF "$(GUI)"=="UNX"
 SHL1OBJS+=$(SLO)$/npunix.obj
 SLOFILES+=$(SLO)$/npunix.obj
+.IF "$(OS)"=="FREEBSD"
+#SHL1STDLIBS+=
+.ELSE
 SHL1STDLIBS+= -ldl -lnsl
-.ENDIF
+.ENDIF # FREEBSD
+.ENDIF # UNX
 .IF "$(GUI)"=="WNT"
 SHL1OBJS+=$(SLO)$/npwin.obj
 SLOFILES+=$(SLO)$/npwin.obj
@@ -123,6 +130,7 @@
 .IF "$(GUI)"=="UNX"
 #APP1STDLIBS+= -lgdk-x11-2.0 -lgtk-x11-2.0
 .IF "$(OS)"=="LINUX"
+.ELIF "$(OS)"=="FREEBSD"
 .ELSE
 APP1STDLIBS+= -ldl -lnsl -lnls -lsocket
 .ENDIF
Index: extensions/source/nsplugin/source/so_env.cxx
===================================================================
RCS file: /cvs/util/extensions/source/nsplugin/source/so_env.cxx,v
retrieving revision 1.2
diff -u -r1.2 so_env.cxx
--- extensions/source/nsplugin/source/so_env.cxx	20 Aug 2004 10:08:00 -0000	1.2
+++ extensions/source/nsplugin/source/so_env.cxx	4 Oct 2004 15:42:46 -0000
@@ -62,7 +62,7 @@
 #ifdef UNIX
 #include <sys/types.h>
 #include <strings.h>
-#ifdef NP_LINUX
+#if defined(NP_LINUX) || defined (FREEBSD)
 #include <stdarg.h>
 #else
 #include <sys/varargs.h>


Index: extensions/source/config/ldap/makefile.mk
===================================================================
RCS file: /cvs/util/extensions/source/config/ldap/makefile.mk,v
retrieving revision 1.2
diff -u -r1.2 makefile.mk
--- extensions/source/config/ldap/makefile.mk	3 Aug 2004 14:39:30 -0000	1.2
+++ extensions/source/config/ldap/makefile.mk	24 Oct 2004 04:05:13 -0000
@@ -97,6 +97,9 @@
         $(CPPUHELPERLIB) \
         $(CPPULIB) \
         $(SALLIB)
+.IF "$(OS)"=="FREEBSD"
+SHL1STDLIBS+=-lcompat
+.ENDIF
 
 DEF1NAME=$(SHL1TARGET)
 DEF1EXPORTFILE=exports.dxp    
